src/main/java/net/minecraftforge/fluids/FluidUtil.java

```diff
@@ -370,6 +372,34 @@ public static IFluidHandlerItem getFluidHandler(@Nonnull ItemStack itemStack)
     }
 
     /**
+     * Gets the fluid handler for a given fluid's block.
+     * The handler's fill method will place a fluid block with the appropriate amount when not simulated.
+     *
+     * @param world the world the block has to be placed in
+     * @param pos   the position the block has to be placed at
+     * @param fluid the fluid to place
+     * @return the fluid block's fluider handler
+     */
+    private static IFluidHandler getPlacementFluidHandler(final World world, final BlockPos pos, Fluid fluid)
+    {
+        final Block block = fluid.getBlock();
+        IFluidHandler handler;
+        if (block instanceof IFluidBlock)
+        {
+            handler = new FluidBlockWrapper((IFluidBlock) block, world, pos);
+        }
+        else if (block instanceof BlockLiquid)
+        {
+            handler = new BlockLiquidWrapper((BlockLiquid) block, world, pos);
+        }
+        else
+        {
+            handler = new BlockWrapper(block, world, pos);
```

Comment:
Given that the wrapper is determined via the fluid's block, you can't get one for a chest unless a fluid considers a chest its block. The method is private on purpose (although I guess it could be public), because the safeguards which check whether the fluid can actually be placed in a spot are all in `FluidUtil.tryPlaceFluid`.